ProcessLib/HydroMechanics/HydroMechanicsFEM.h

```diff
@@ -354,6 +416,181 @@ class HydroMechanicsLocalAssembler : public ProcessLib::LocalAssemblerInterface
         }
     }
 
+    void postTimestepConcrete(std::vector<double> const& local_x) override
+    {
+        double const& t = _process_data.t;
+
+        auto p =
+            Eigen::Map<typename ShapeMatricesTypePressure::template VectorType<
```

Comment:
The `MathLib::toVector(vector, size)` is not applicable here, because the data is located at the offset `pressure_index` and is not the whole vector...
One could add the additional parameters and it would look like this (on the caller side):
```c++
auto p =
            MathLib::toVector<typename ShapeMatricesTypePressure::template VectorType<
                pressure_size> const>(local_x, pressure_index, pressure_size);
```
which is only one character shorter then the current version....